src/components/Nurseries/index.tsx

```diff
@@ -1,5 +1,159 @@
+import { Box, Grid, Typography } from '@mui/material';
+import { Button, Table, TableColumnType } from '@terraware/web-components';
+import TextField from '@terraware/web-components/components/Textfield/Textfield';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { search, SearchNodePayload } from 'src/api/seeds/search';
+import { Facility } from 'src/api/types/facilities';
+import { APP_PATHS } from 'src/constants';
 import strings from 'src/strings';
+import { ServerOrganization } from 'src/types/Organization';
+import { getAllNurseries } from 'src/utils/organization';
+import { getRequestId, setRequestId } from 'src/utils/requestsId';
+import useDebounce from 'src/utils/useDebounce';
+import TfMain from '../common/TfMain';
+import PageSnackbar from '../PageSnackbar';
 
-export default function NurseriesList(): JSX.Element {
-  return <p>{strings.NURSERIES}</p>;
+type NurseriesListProps = {
+  organization: ServerOrganization;
+};
+
+const columns: TableColumnType[] = [
+  { key: 'name', name: 'Name', type: 'string' },
+  { key: 'description', name: 'Description', type: 'string' },
+];
+
+export default function NurseriesList({ organization }: NurseriesListProps): JSX.Element {
+  const { isMobile } = useDeviceInfo();
+  const history = useHistory();
+  const [nurseries, setNurseries] = useState<Facility[]>();
+  const [temporalSearchValue, setTemporalSearchValue] = useState('');
+  const debouncedSearchTerm = useDebounce(temporalSearchValue, 250);
+  const [results, setResults] = useState<Facility[]>();
+
+  useEffect(() => {
+    const getNurseries = () => {
+      if (organization) {
+        const orgNurseries = getAllNurseries(organization).filter((n) => n !== undefined) as Facility[];
+        setNurseries(orgNurseries);
+        setResults(orgNurseries);
+      }
+    };
+
+    getNurseries();
+  }, [organization]);
+
+  const goToNewNursery = () => {
+    const newNurseryLocation = {
+      pathname: APP_PATHS.NURSERIES_NEW,
+    };
+    history.push(newNurseryLocation);
+  };
+
+  const clearSearch = () => {
+    setTemporalSearchValue('');
+  };
+
+  const onChangeSearch = (id: string, value: unknown) => {
+    setTemporalSearchValue(value as string);
+  };
+
+  useEffect(() => {
+    const refreshSearch = async () => {
```

Comment:
Something on my mind earlier from other pages like seed banks etc. we could do in memory search if we wanted - no BE request.
BE request is fine.. but we should fetch latest results on line 107 (otherwise we could technically end up showing latest results only when there's a search term - otherwise we show stale info that was fetched on page load).

Reply:
my suggestion is to remove the `if` on line 64 and remove the `else` section from line 106 - so we are always fetching latest results. that would be a simple fix. thoughts?

Reply:
But in that case we would be always doing a BE search, not a search in memory, right?

Reply:
Yes - that's if we want to use BE for search.
We could always use in-memory or always use BE for search. If we use a mix of that - results may be inconsistent.
BE search should be better so we always show latest results?
My suggestion may lead to infinite useEffect trigger (because nurseries keeps changing) - we can use a lodash isEqual check before setting nursery results (don't set if the results in state are same as what we are trying to set).
My main suggestion is avoiding a mix of BE vs in-memory.

Reply:
We can revisit this later if you prefer - since this is how the other pages behave.

Reply:
I tested using only the search api (not getting the nurseries from the org anytime) and it works correctly! This way would always show latests results

Reply:
> I tested using only the search api (not getting the nurseries from the org anytime) and it works correctly! This way would always show latests results

Thanks, looks good on vercel. Sorry for the trouble.